### PR TITLE
PathToTina Documentation

### LINKED ISSUE #1170: Document customizing the "Path to Tina" property
Document the path to Tina property.

### PR DESCRIPTION
**Pull request by jhuggett**
closes #1170

### CODE REVIEW

File: content/docs/tina-cloud/faq.md [hunk 1]
```diff
@@ -49,15 +48,27 @@ A fair use policy will be coming soon.
...
+
+It does! Tina Cloud can work with sites inside monorepos by configuring the path to your .tina folder in your Projects.
+
+If your repo isn't a monorepo, then there's no need to do any configuration. We'll look for your Tina Config at the root of your repo.
+
+Here's an example monorepo structure that works with Tina Cloud:
+
+    /projects/site-a
+    /projects/site-b
+    /projects/site-c
```
Comment (kldavis4) [inline content/docs/tina-cloud/faq.md:61]: format as code [block](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks)?

File: content/docs/tina-cloud/faq.md [hunk 2]
```diff
@@ -49,15 +48,27 @@ A fair use policy will be coming soon.
 
 We will contact you if we believe your use case may eventually fit within our post-beta paid plans.
 
+## **Does Tina Cloud work with Monorepos?**
+
+It does! Tina Cloud can work with sites inside monorepos by configuring the path to your .tina folder in your Projects.
```
Comment (kldavis4) [inline content/docs/tina-cloud/faq.md:53]:
```suggestion
It does! Tina Cloud can work with sites inside monorepos by specifying the path to your `.tina` folder in your Project configuration.
```

File: content/docs/tina-cloud/faq.md [hunk 3]
```diff
@@ -49,15 +48,27 @@ A fair use policy will be coming soon.
...
+
+If your repo isn't a monorepo, then there's no need to do any configuration. We'll look for your Tina Config at the root of your repo.
```
Comment (kldavis4) [inline content/docs/tina-cloud/faq.md:55]:
```suggestion
If your repo is not a monorepo, there's no need to do any configuration. We'll expect your `.tina` folder at the root of your repo.
```

**Comment by jhuggett** [comment]
added code block

File: content/docs/tina-cloud/faq.md (hunk 1 above)
Comment (jhuggett) [reply content/docs/tina-cloud/faq.md:61]: fixed

**Review by kldavis4 — APPROVED** [review]

**Pull request merged**